documentation/release-notes/topics/version_4.rst

```diff
@@ -2,6 +2,74 @@
+   2. Support for the *Windows* version of the Data Collector will end on *January 31, 2025*. Customers are encouraged to switch to the Linux version before then. Detailed :ref:`installation instructions <Setting-up-data-collectors-doc>` are in the product documentation.
+   3. Microsoft has `announced <https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/>`_ the sudden and quick retirement of the Webhook interface within Microsoft Teams. This impacts the Notifications Channels functionality. We are evaluating options and will be addressing this in an upcoming release.
+
+========================================
+Enhanced Feature: Alarm event management
+========================================
+The alarm grid has been improved to show the alarm event category.
+
+===========================================
+Enhanced Feature: Alarm event notifications
+===========================================
+Alarm event categories have been added as a criteria for a alarm events. A Hyperview administrator is now able to select a specific alarm event category or all catagories when defining a policy.
+
+Alarm even category options are:
+
+- All Categories
+- AssetTracker Status
+- Data Collector Status
+- Firmware Status
+- Host Unreachable
+- Sensor Threshold Check
+- SNMP Trap
+- Unknown Trap
+
+Existing notification policies will have the "All Categories" option selected by default. This can be changed by editing the policy.
+
+=======================================================
+Enhanced Feature: Asset alarm event notification muting
+=======================================================
+Assets undergoing maintenance or causing excessive notifications can now be muted for a specific period of time. The functionality is available under the **Actions -> Mute Notifications** menu.
+
+A Mute action can be cancelled from the **Actions -> Cancel Mute Notifications** menu.
+
+================================================================
+Enhanced Feature: Bulk actions -> Mute/Cancel Mute Notifications
+================================================================
+Mute and Cancel Mute Notifications can also be performed as a bulk action on a list of assets. The action is available from Advanced Search, Information -> Assets and the Assets By Type Bulk Actions menu.
+
+===================================
+Enhanced Feature: Dashboard widgets
+===================================
+A widget was added to display asset documents and links.
+
+======================================
+Other notable changes and improvements
+======================================
+- The asset Actions menu has been enhanced to group related actions.
+- Server assets types can bow be changed into Storage types and ViceVersa.
```

Comment:
"Server asset types..." instead of "Server assets types..."

Reply:
Thanks, Addressed.